### PR TITLE
Product Gallery Large Image: Add the php experimental flag

### PR DESCRIPTION
**Pull request by danieldudzic**
Add PHP experimental flag to Product Gallery Large Image block

Reference: https://github.com/woocommerce/woocommerce-blocks/pull/10141

### CODE REVIEW

**Review by thealexandrelara — APPROVED** [review]
Nice! LGTM! ✅

File: src/BlockTypesController.php [hunk 1]
```diff
@@ -229,7 +228,8 @@ protected function get_block_types() {
 			$block_types[] = 'ProductCollection';
 			$block_types[] = 'ProductRatingStars';
 			$block_types[] = 'ProductTemplate';
-			$block_types[] = 'ProductGallery';
+      $block_types[] = 'ProductGallery';
+			$block_types[] = 'ProductGalleryLargeImage';
```
Comment (thealexandrelara) [inline src/BlockTypesController.php]: The indentation is incorrect in these lines

**Review by thealexandrelara — CHANGES_REQUESTED** [review]
I just left a small comment about the PHP CS error that is being shown

File: src/BlockTypesController.php (hunk 1 above)
Comment (danieldudzic) [reply src/BlockTypesController.php]: Thanks!

**Review by thealexandrelara — APPROVED** [review]
Great! Thank you for addressing that point! ✅

**Pull request merged**